libs/wingsdk/src/util.ts

```diff
@@ -161,3 +162,16 @@ export function sanitizeCodeText(code: string): string {
 export function sanitizeCode(code: Code): string {
   return sanitizeCodeText(code.text);
 }
+
+export async function runCommand(cmd: string): Promise<any> {
+  const raw = await new Promise((resolve, reject) => {
+    exec(cmd, (error, stdout, stderr) => {
```

Comment:
I suspect this might be a security hole, also considering we're passing the `REDIS_IMAGE` env var as an arg. Suggest switching to `execFile` and changing the api of `runCommand` accordingly (`(cmd: string, args: string[])`).

Reply:
I switched this to `execFile` but curious @yoav-steinberg what is the security hole concerns?

Reply:
afaict, `REDIS_IMAGE` is not an environment variable, so there shouldn't be a concern with shell injection

Reply:
@MarkMcCulloh I actually just changed it to allow it as an environment variable. But even at that I think we should be fine, this runCommand() helper would only ever be run during local compilation. With that said I still think going with args[] is good.